packages/trip-details/i18n/fr.yml

```diff
@@ -7,6 +7,10 @@ otpUi:
       {walkMinutes, plural, one {<strong># minute</strong> effectuée} other {<strong># minutes</strong> effectuées}} à pied et
       {bikeMinutes, plural, one {<strong># minute</strong> effectuée} other {<strong># minutes</strong> effectuées}} en vélo sur ce trajet.
       (d'après <dietaryLink>Dietary Guidelines for Americans 2005, page 16, Table 4</dietaryLink>)
+    co2: "CO<sub>2</sub> émis: <strong>{co2}</strong>"
+    co2description:
```

Comment:
Should this be translated?

Reply:
Done in https://github.com/opentripplanner/otp-ui/pull/345/commits/1afba88c57b910c63040c7eed23e5b5ff64b1a6a.